core/framework/extractors/docling_extractor_backup_20250913.py

```
@@ -0,0 +1,403 @@
+        
+        # Check file type and handle appropriately
+        pdf_path_obj = Path(pdf_path)
+        
+        # For text files, handle directly (useful for testing)
+        if pdf_path_obj.suffix in ['.txt', '.text', '.md']:
+            try:
+                with open(pdf_path, 'r', encoding='utf-8') as f:
+                    text = f.read()
+                
+                if not text.strip():
+                    raise RuntimeError(f"Text file is empty: {pdf_path}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)